tests/unit/amo/components/TestLink.js

```diff
@@ -205,4 +205,14 @@ describe(__filename, () => {
 
     expect(root.find(Icon)).toHaveProp('name', `external-dark`);
   });
+
+  it('adds rel="noopener noreferrer" when target is "_blank"', () => {
+    const root = render({
+      href: '/test',
+      target: '_blank',
+      rel: 'noopener noreferrer',
+    });
+
+    expect(root.find('a')).toHaveLength(1);
```

Comment:
That looks ok to me, but I'd rather test that the link component has `target` prop and its corresponding `rel` prop:

```js
const root = render({ href: '/', target: '_blank' });

expect(root.find('a')).toHaveProp('target', '_blank');
expect(root.find('a')).toHaveProp('rel', 'noopener noreferrer');
```

Reply:
You can also add another test case with `target: 'not-blank'`. You should expect the link to have a prop `target` but no `rel` prop.

Reply:
Changed the test as above  -
```
  it('adds rel="noopener noreferrer" when target is "_blank"', () => {
    const root = render({ href: '/', target: '_blank' });
    
    expect(root.find('a')).toHaveProp('target', '_blank');
    expect(root.find('a')).toHaveProp('rel', 'noopener noreferrer');
  });
```

But giving error - 
![screen shot 2018-08-22 at 1 13 51 pm](https://user-images.githubusercontent.com/17110163/44449970-452a8400-a60d-11e8-89eb-30b8081ec3f1.png)

Reply:
Should i remove **rel="noopener noreferrer"** from various Component files - 
```
PermissionsCard, ContributeCard, discoPurify, FooterBase, DiscoPaneBase 
```
and also the tests from test files -
```
test_utils, TestAddons, TestLink
```

Reply:
Should i remove **rel="noopener noreferrer"** from various Component files - 
```
PermissionsCard, ContributeCard, discoPurify, FooterBase, DiscoPaneBase 
```
and also the tests from test files -
```
test_utils, TestAddons, TestLink
```

Reply:
@nabendu82 yes, you can remove rel="noopener noreferrer" from all components in `src/amo/` but I don't think you should change `discoPurify` and other files in `src/disco/`.